src/dos/dos.cpp

```diff
@@ -178,6 +178,22 @@ void DOS_SetCountry(uint16_t country_number)
 	dos.tables.country[DOS_DECIMAL_SEPARATOR_OFS]   = country_info.decimal_separator;
 }
 
+uint16_t DOS_GetCurPackedTime()
+{
+	uint32_t ticks   = mem_readd(BIOS_TIMER);
+	uint32_t seconds = (ticks * 10) / 182;
```

Comment:
What is 182? Please extract all magic constants into `constexpr` with descriptive names.

Reply:
I have no idea, I just copied the code from drive_local.cpp.

Reply:
Yeah that's exactly the problem, see 😄 It will do then, but if you could do some detective work and figure out the meaning, that would be nice.